package.json

```diff
@@ -1,40 +1,73 @@
 {
-    "name": "passr",
-    "version": "0.1.0",
-    "private": true,
-    "dependencies": {
-        "@testing-library/jest-dom": "^4.2.4",
-        "@testing-library/react": "^9.5.0",
-        "@testing-library/user-event": "^7.2.1",
-        "@types/jest": "^24.9.1",
-        "@types/node": "^12.12.62",
-        "@types/react": "^16.9.49",
-        "@types/react-dom": "^16.9.8",
-        "react": "^16.13.1",
-        "react-dom": "^16.13.1",
-        "react-scripts": "3.4.3",
-        "typescript": "^3.7.5"
-    },
-    "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject",
-        "lint": "echo 'Implement linting!'"
-    },
-    "eslintConfig": {
-        "extends": "react-app"
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    }
+	"name": "passr",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@testing-library/jest-dom": "^4.2.4",
+		"@testing-library/react": "^9.5.0",
+		"@testing-library/user-event": "^7.2.1",
+		"@types/jest": "^24.9.1",
+		"@types/node": "^12.12.62",
+		"@types/react": "^16.9.49",
+		"@types/react-dom": "^16.9.8",
+		"@typescript-eslint/eslint-plugin": "^2.34.0",
+		"@typescript-eslint/parser": "^2.34.0",
+		"@typescript-eslint/typescript-estree": "^2.34.0",
+		"eslint-config-prettier": "^6.14.0",
+		"eslint-plugin-prettier": "^3.1.4",
+		"eslint-plugin-react": "^7.19.0",
+		"eslint-plugin-react-hooks": "^1.7.0",
+		"react": "^16.13.1",
+		"react-dom": "^16.13.1",
+		"react-scripts": "3.4.3",
+		"typescript": "^3.7.5"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"test": "react-scripts test",
+		"eject": "react-scripts eject",
+		"eslint": "eslint 'src/**/*.{t,j}s{x,}'",
+		"eslint:fix": "eslint 'src/**/*.{t,j}s{x,}', --fix'",
+		"prettier": "prettier --write 'src/**/*.{t,j}s{x,}'"
+	},
+	"prettier": {
+		"useTabs": true,
+		"tabWidth": 4,
+		"semi": true,
+		"singleQuote": true,
+		"proseWrap": "never"
+	},
+	"eslintConfig": {
+		"extends": "react-app"
+	},
```

Comment:
I think we should use a separate [prettier configuration] file, like `.prettierrc.js`because I don't want to over bloat our package.json. It will make prettier changes easier to find in the future too. 

Same applied for the eslintConfig. I think you can add this to the .eslintrc.js